.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "website/themes/docsy"]
+	path = website/themes/docsy
+	url = https://github.com/google/docsy.git
```

Comment:
Maybe it's just me, but I always have all sorts of problems with git submodules. Is there any way to avoid using one for docsy?

Reply:
I think thats pretty common. @jmrodri mentioned that he has added themes by carrying everything and updating it manually. I am thinking of that as a backup plan if submodules dont work out, but submodules are what docsy suggests.

Reply:
@joelanford @asmacdo yeah I *despise* git submodules. They always feel out of sync etc. I know this is very common for the hugo site. All of their tutorials show to use them for themes. :man_shrugging: